release/create_release_charts.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
Where do you call this file in release process?

Reply:
From create_release_archives I think.